integrationtest/mongodb/src/test/resources/arquillian.xml

```diff
@@ -1,28 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
yes .. it always was .. with some luck :)
as I say on the JIRA, there was a duplicate resource on classpath, making it pseudo-random behaviour. Luckily most of these resource were dubplicate in content too, but not all of them.